raiden/api/rest.py

```diff
@@ -481,6 +483,14 @@ def get_our_address(self) -> Response:
     def get_raiden_version(cls) -> Response:
         return api_response(result=dict(version=get_system_spec()["raiden"]))
 
+    def get_node_settings(self) -> Response:
+        settings = dict(pathfinding_service_address="")
+        pfs_config = self.raiden_api.raiden.config.pfs_config
+        if pfs_config is not None:
+            settings["pathfinding_service_address"] = pfs_config.info.url
```

Comment:
Wouldn't `None` be a better value for the absence of a PFS?

I would have expected just
```suggestion
        settings = dict(
            pathfinding_service_address=self.raiden_api.raiden.config.pfs_config
        )
```

Reply:
Yes, I agree that it makes more sense. 

The `/contracts` endpoint also returns empty strings in case there are no services contracts. Maybe I change that too?

Reply:
In a strict sense, that is a breaking change. But I think we can get away with it in this case, so go for it!